guides/common/modules/proc_importing-kickstart-repositories.adoc

```diff
@@ -1,107 +1,101 @@
 endif::[]
-
 . When the download completes, copy the ISO image to {ProjectServer}.
 . On {ProjectServer}, create a mount point and temporarily mount the ISO image at that location:
 +
-[options="nowrap" subs="+quotes"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # mkdir /mnt/_iso_
 # mount -o loop _rhel-binary-dvd.iso_ /mnt/_iso_
 ----
-
-ifeval::["{rhel-version}" == "8"]
-. Create directories for {RHEL} 8 AppStream and BaseOS Kickstart repositories:
+ifeval::["{el-major-version}" >= "8"]
+. Create directories for {RHEL} {el-major-version} AppStream and BaseOS Kickstart repositories:
 +
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# mkdir --parents /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/appstream/kickstart
-# mkdir --parents /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/baseos/kickstart
+# mkdir --parents /var/www/html/pub/{Project}-import/content/dist/rhel{el-major-version}/{el-major-version}.{el-minor-version}/x86_64/appstream/kickstart
+# mkdir --parents /var/www/html/pub/{Project}-import/content/dist/rhel{el-major-version}/{el-major-version}.{el-minor-version}/x86_64/baseos/kickstart
 ----
 endif::[]
 
-ifeval::["{rhel-version}" == "7"]
+ifeval::["{el-major-version}" == "7"]
 . Create Kickstart directories:
 +
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# mkdir --parents /var/www/html/pub/sat-import/content/dist/rhel/server/7/7.7/x86_64/kickstart/
+# mkdir --parents /var/www/html/pub/{Project}-import/content/dist/rhel/server/7/7.7/x86_64/kickstart/
 ----
 endif::[]
 
 . Copy the `kickstart` files from the ISO image:
 +
-ifeval::["{rhel-version}" == "8"]
-[options="nowrap" subs="+quotes"]
+ifeval::["{el-major-version}" >= "8"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# cp -a /mnt/_iso_/AppStream/* /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/appstream/kickstart
-
-# cp -a /mnt/_iso_/BaseOS/* /mnt/_iso_/images/ /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/baseos/kickstart
+# cp -a /mnt/_iso_/AppStream/* /var/www/html/pub/{Project}-import/content/dist/rhel{el-major-version}/{el-major-version}.{el-minor-version}/x86_64/appstream/kickstart
+# cp -a /mnt/_iso_/BaseOS/* /mnt/_iso_/images/ /var/www/html/pub/{Project}-import/content/dist/rhel{el-major-version}/{el-major-version}.{el-minor-version}/x86_64/baseos/kickstart
 ----
 +
 Note that for BaseOS, you must also copy the contents of the `/mnt/_iso_/images/` directory.
 endif::[]
-ifeval::["{rhel-version}" == "7"]
-[options="nowrap" subs="+quotes"]
+ifeval::["{el-major-version}" == "7"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# cp -a /mnt/_iso_/* /var/www/html/pub/sat-import/content/dist/rhel/server/7/7.7/x86_64/kickstart/
+# cp -a /mnt/_iso_/* /var/www/html/pub/{Project}-import/content/dist/rhel/server/7/7.7/x86_64/kickstart/
```

Comment:
```suggestion
# cp -a /mnt/_iso_/images/* /var/www/html/pub/{Project}-import/content/dist/rhel/server/7/7.7/x86_64/kickstart/
```
I am not a developer so I can't confirm this, but the step description says to copy contents of the `/mnt/iso/images` directory.